vms/platformvm/service.go

```diff
@@ -1984,15 +1943,8 @@ func (s *Service) chainExists(ctx context.Context, blockID ids.ID, chainID ids.I
 		}
 	}
 
-	tx, _, err := state.GetTx(chainID)
-	if err == database.ErrNotFound {
-		return false, nil
-	}
-	if err != nil {
-		return false, err
-	}
-	_, ok = tx.Unsigned.(*txs.CreateChainTx)
-	return ok, nil
+	_, err := state.GetChainSubnet(chainID)
+	return err == nil, err
```

Comment:
I think this is a behavior change. Previously, if `GetTx` returned `database.ErrNotFound`, we'd return `nil`. Now we return `database.ErrNotFound`